sdk/tests/conformance2/samplers/samplers.html

```diff
@@ -126,6 +133,72 @@
     s1 = null;
 }
 
+function runParameterTest() {
+    // TODO: Add params checking cases.
+    debug("Testing getSamplerParameter and samplerParameter[if]");
+
+    s = gl.createSampler();
+    gl.bindSampler(0, s);
+
+    debug("Checking samplerParameteri");
+    testCases = [
+      { pname: gl.TEXTURE_WRAP_S,
+        param: gl.REPEAT,
+        expectedError: gl.NO_ERROR },
+      { pname: gl.TEXTURE_MIN_FILTER,
+        param: gl.LINEAR,
+        expectedError: gl.NO_ERROR },
+      { pname: gl.TEXTURE_COMPARE_MODE,
+        param: gl.NONE,
+        expectedError: gl.NO_ERROR },
```

Comment:
Could you please also test valid pname TEXTURE_WRAP_R, TEXTURE_WRAP_T, TEXTURE_MAG_FILTER, TEXTURE_COMPARE_FUNC? The list is short, so it's good to verify them all.